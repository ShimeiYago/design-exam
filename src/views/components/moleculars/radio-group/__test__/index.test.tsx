import { shallow, ShallowWrapper } from 'enzyme';
import { RadioGroup, Props } from '..';

let wrapper: ShallowWrapper<Props>;

const props: Props = {
  options: [
    { label: 'schooling', value: '1' },
    { label: 'employment', value: '2' },
  ],
  groupLabel: 'classification',
  selectedValue: '1',
  onChange: jest.fn(),
};

describe('Shallow Snapshot Tests', () => {
  beforeEach(() => {
    wrapper = shallow(<RadioGroup {...props} />);
  });

  it('basic', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
