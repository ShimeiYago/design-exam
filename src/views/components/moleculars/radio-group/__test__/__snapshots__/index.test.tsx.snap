// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests basic 1`] = `
<div
  className="group"
>
  <Spacing
    marginBottom="8px"
  >
    <TextWrapper
      bold={true}
      size="small"
    >
      classification
    </TextWrapper>
  </Spacing>
  <label
    className="label"
    key="radio-1"
  >
    <input
      checked={true}
      className="radioInput"
      onChange={[MockFunction]}
      type="radio"
      value="1"
    />
    schooling
  </label>
  <label
    className="label"
    key="radio-2"
  >
    <input
      checked={false}
      className="radioInput"
      onChange={[MockFunction]}
      type="radio"
      value="2"
    />
    employment
  </label>
</div>
`;
